zephyr/src/utils.c

```diff
@@ -675,6 +675,33 @@ void set_netmask(char *ifname) {
 }
 
 int find_interface_ip(char *ipaddr, int ipaddr_len, char *name) {
+    char tmp[32];
+
+    if (strcmp(name, get_wireless_interface()))
+        return 0;
+
+    const struct device *dev = device_get_binding(name);
+    struct net_if *wifi_iface = net_if_lookup_by_dev(dev);
+
+    if (net_if_is_wifi(wifi_iface)) {
+#ifdef CONFIG_NET_IPV4
```

Comment:
What if both IPV4 and IPV6 are enabled and the interface only has v6 or v4?

Reply:
Cleaned up reading IPv6 address changes, as QT has only IPv4 support tests.